source/globalCommands.py

```diff
@@ -3483,8 +3523,74 @@ def script_braille_scrollForward(self, gesture):
 		description=_("Routes the cursor to or activates the object under this braille cell"),
 		category=SCRCAT_BRAILLE
 	)
-	def script_braille_routeTo(self, gesture):
+	def script_braille_routeTo(self, gesture: inputCore.InputGesture) -> None:
+		"""Routes cursor to or activates the object under this braille cell."""
+		# Try to ensure that braille message can be dismissed.
+		if braille.handler.buffer is braille.handler.messageBuffer:

```

Comment:
Please move this below the call of `braille.handler.routeTo(gesture.routingIndex)` so you no longer have to duplicate that line.

Reply:
If pressing routing button causes message it appears again when dismissing with routing button. Reverting this at least for now.